google-cloud-bigtable-deps-bom/pom.xml

```diff
@@ -79,7 +79,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-shared-dependencies</artifactId>
-        <version>0.7.0</version>
+        <version>0.8.0</version>
```

Comment:
this is already up to 0.8.1, so I think if you rebase to the latest you should be all set for this dep: https://github.com/googleapis/java-bigtable/commit/637350b4cc034f603cde3a18c0a73f7cc6cc0f82

Reply:
Thanks, done